index.bs

```diff
@@ -94,13 +101,13 @@ ISSUE: Should this specification also define a structural brand stricter than 'm
 
 <h3 id="image-item">AV1 Image Item</h3>
 
-<p>When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>, it is called an <dfn>AV1 Image Item</dfn>, and SHALL obey the following constraints:
-<ul>
-<li>The item data SHALL be the content of a single valid [=AV1 Sync Sample=] as defined in [[!AV1-ISOBMFF]] with the additional constraints that there SHALL be only one [=Sequence Header OBU=] and it SHALL have its <code>[=still_picture=]</code> flag set to 1 and SHOULD have its <code>[=reduced_still_picture_header=]</code> flag set to 1.
-<li>The item SHALL be associated with an [=AV1 Item Configuration Property=].</li>
-</ul>
-
-</p>
+When an image item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>, it is called an <dfn>AV1 Image Item</dfn>, and shall obey the following constraints:
+    
+  - The [=AV1 Image Item=] is the content of a single valid [=AV1 Sync Sample=] as defined in [[!AV1-ISOBMFF]].
```

Comment:
There is a difference between the item (which is a logical entity) vs. the item data which is the physical one. The use of item data here was meant to emphasize that the data has to be physically formed by the content of sync sample.

Reply:
There are at least 3 different terms:
- "image" as defined in HEIF, i.e. arrays of pixels. This is a logical term used for rendering purposes
- "image item", i.e. a logical term used for storage purposes.  "image item" and "image" are very similar. I think an image item may carry multiple images in the case of scalability.
- 'item data", i.e. the actual bytes stored in the 'idat' or 'mdat' box. 
I prefer to use "item data" in this sentence.

Reply:
I understand the distinction, but the problem is that "item data" is not a defined term in any of the specs that this spec references. And introducing a new concept of "item data" in this spec would seem odd. I see that for HEVC, the HEIF spec defines a struct called "HEVCItemData", which is exactly that we would need for AV1. 

Looking at the AV1-ISOBMFF spec, the nearest equivalent to "HEVCItemData" is "AV1 Sample". The definition about "AV1 Sample" is a bit muddled up because it gives the defintion of the item data and talks about Track header all in the same section. But I think it works for our purposes.

The AVIF spec currently does not use "AV1 Sample" but rather uses "AV1 Sync Sample", which is not a defined term in the AV1-ISOBMFF spec.

I will change this section (2.1) to use "AV1 Sample" and you can review again and let me know if looks OK.